testData/pathfinding.data.js

```diff
@@ -0,0 +1,393 @@
+import { BuildingId } from "../src/types/main";
+
+export const h85103toH815 = [
```

Comment:
Is there a reason this is a js file instead of typescript?

Reply:
No, will change it to ts.